setup.py

```diff
@@ -106,21 +114,20 @@ def build_extensions(self):
 
     return ConditionalOpenMP
 
-
+# Grabbing __version__
 with open('skimage/__init__.py', encoding='utf-8') as fid:
     for line in fid:
         if line.startswith('__version__'):
             VERSION = line.strip().split()[-1][1:-1]
             break
 
-
+# Parsing requirements file
```

Comment:
Is this comment necessary? The function name contains the exact same English root words.
```suggestion
```